packages/@webex/plugin-meetings/src/constants.ts

```diff
@@ -237,6 +237,14 @@ export const CALENDAR_EVENTS = {
   DELETE: 'event:calendar.meeting.delete',
 };
 
+// Specificied here: https://confluence-eng-gpk2.cisco.com/conf/display/LOCUS/Reclaim+Host+Role#:~:text=Note-,2400127,-Reclaim%20Host%20Role
```

Comment:
Could we remove this link from the code comments please? This is internal information that we shouldn't expose to external developers.